api/config.js

```diff
@@ -75,6 +76,15 @@ config.auth.secret = config.secret;
 config.auth.header = 'Authorization';
 config.auth.expiration = '7d';
 
+// follows enums defined in v1/utils/scopes.js
+// 7 days in milliseconds
+const MILLISECONDS_PER_HOUR = 3600000;
+const HOURS_PER_DAY = 24;
+const DAYS_PER_WEEK = 7;
+config.tokenExpiration.AUTH = DAYS_PER_WEEK * HOURS_PER_DAY * MILLISECONDS_PER_HOUR;
+config.tokenExpiration.OTHER = DAYS_PER_WEEK * HOURS_PER_DAY * MILLISECONDS_PER_HOUR;
+config.tokenExpiration.DEFAULT = DAYS_PER_WEEK * HOURS_PER_DAY * MILLISECONDS_PER_HOUR;
```

Comment:
Maybe declare this `DEFAULT` property first and then say `config.token.expiration.AUTH = config.token.expiration.DEFAULT` (or declare a local `defaultTokenExpiration` and set it to that instead).